RELEASE.md

```diff
@@ -63,8 +63,14 @@ git push --tags
 To provide a signed tarball for distributions we use sdist and gpg:
 
 ```
-python setup.py sdist
-gpg -u markus@lazyfrosch.de --armor \
+VERSION=$(git describe --tags | sed s/^v//)
+GPGKEY=markus@lazyfrosch.de
```

Comment:
would something like `GPGKEY=$(git config --get user.email)` work for you?  If you're in the terminator checked out directory it should pull your correct email.  That way we can both just copy and paste from the document.

Reply:
Nice idea, done